### PR TITLE
modify README.md

### PR DESCRIPTION
**Pull request by andela-sjames**
#41

### CODE REVIEW

**Comment by andela-sjames** [comment]
@andela-kadeniyi  please review updated README.md

File: README.md [hunk 1]
```diff
@@ -31,39 +54,32 @@ DAEMON: False
...
+**Run the following command to create the tables **
+```
+python django_foodbot/manage.py makemigrations
 ```
-**Download the database backup from heroku:**
+```
+python django_foodbot/manage.py migrate
+```
+
+###Get the latest food-bot database from heroku stage.
```
Comment (andela-kadeniyi) [inline README.md]: @andela-sjames I'm not sure `heroku stage` is easy to understand. Do you mean `foodbot staging app` or something else.

File: README.md [hunk 2]
```diff
@@ -31,39 +54,32 @@ DAEMON: False
...
+
+First we will need to add you as a collaborator on heroku
+
+**To download the database backup from heroku:**
+**Note:** You will need to be added as a collaborator on heroku staging
```
Comment (andela-kadeniyi) [inline README.md]: @andela-sjames @andela-kanyanwu How can I go about this without downloading the database backup from heroku. Can we have a link to always download a database template from?

File: README.md [hunk 3]
```diff
@@ -80,30 +96,6 @@ You can click [here](https://devcenter.heroku.com/articles/heroku-postgres-impor
 
 **We will work on creating a script that would do the setup automatically for you but for now, please follow these steps and let any of the collaborators know if you have any issues.**
```
Comment (andela-kadeniyi) [inline README.md]: @andela-kanyanwu How come we still have this line even when the script is no longer available?

**Comment by andela-kadeniyi** [comment]
@andela-sjames @andela-kanyanwu I'm done reviewing. Had a few comments

**Pull request merged**